packages/mcl/src/src/mcl/utils/coda.d

```diff
@@ -527,6 +527,16 @@ struct CodaApiClient
         coda.deleteRow("dEJJPwdxcw", tables[0].id, resp[0]);
     }
 
+    void updateOrInsertRow(string docId, string tableId, RowValues values) {
+        auto table = listRows(docId, tableId);
+        auto rows = find!(row => row.name == values.cells[0].value)(table);
+        if (rows.length > 0) {
+            updateRow(docId, tableId, rows[0].id, values);
+        }
+        else {
+            insertRows(docId, tableId, [values]);
+        }
```

Comment:
If there is conditional (ternary) operator in `d` here might be nice place to use it

Reply:
Except the args aren't the same. One requires the row id, the other doesn't.
Also, wait a minute, for me it's a helper function called updateOrInsertRow. I think I forgot to push.